### DISCUSSION TITLE
Loop switcher

### DISCUSSION THREAD

**Discussion by DWSQUIRES** [alf45tar/PedalinoMini#239] [Q&A] [answered]
Love this project, have parts on order to build 3!
Quick ballpark question from a newb. 

Would it be possible to use a similar hardware setup to  make a midi triggered loop switcher so that regular stompboxes can be activated via the midi switches of the pedalinomini? Also, would there be any delay in transmitting these messages over Bluetooth? 

Any help appreciated 👍
Dan

**Comment by alf45tar** [comment, marked as answer]
Not sure to understand the question. PedalinoMini can be used to trigger any midi device including loop station.

**Reply by DWSQUIRES** [reply]
Sorry I will try to clarify.  I would like to build a loop switcher (not an audio looper station, the terms are confusing haha) with similar hardware to the pedalinomini (mobile interface, ble, WiFi, addressable LEDs) Obviously I will need to include relays and a bunch of input and output jacks but was wondering if it was possible.

**Reply by alf45tar** [reply]
PedalinoMini is a good starting point also for a loop switcher. Which relays are you planning to use? Electromechanical or solid state? I think Bluetooth is much much much faster than any electromechanical relay.

**Reply by DWSQUIRES** [reply]
Good to know. I have a few Omron G6K series relays (G6K-2P-5VDC) so was going to try with those.
Could Pedalino be set to receive a MIDI message and close a relay? (via a mosfet)

**Reply by alf45tar** [reply]
Yes, of course with small code changes.

**Reply by SteloPin** [reply]
Great idea to add relay output (on, off or just a short pulse) to the project. E.g. I have an A/B Novel switcher which I could control then by midi. I build an arduino sketch some years ago for this but without bluetooth.

**Comment by Designbase** [comment]
Have a look at this one: https://github.com/brianthornock/MLS_5 
https://scientificguitarist.wixsite.com/home/mls-5

**Comment by SteloPin** [comment]
I would like to bring this up. Can you to add a relay output (on, off or just a short pulse) when receiving a configurable MIDI message?
In my case I have a A/B Novel switcher to switch between 2 Guitars. My MIDI-Program will send the messages between songs.
Lets say I could configure two midi messages to listen to. 1 to make a pulse to switch to A, and 2 to make a pulse to switch to B.
I only want to get a short pulse if midi message 1 changes to midi message 2 and visa verse. If the same message is received multiple times (because I may have several songs, which will be played with the same guitar), no pulse will be sended.
